Vagrantfile

```diff
@@ -0,0 +1,23 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"

```

Comment:
(Assuming you're running this via virtualbox and can forward the ports) you'll want to add a line here for port forwarding:
```
config.vm.network "forwarded_port", guest: 5000, host: 5000
```